query/src/main/java/org/infinispan/query/backend/IndexModificationStrategy.java

```diff
@@ -39,9 +39,13 @@ public boolean shouldModifyIndexes(FlagAffectedCommand command, InvocationContex
       @Override
```

Comment:
same thing happens on `LOCAL_ONLY`, because `PRIMARY_OWNER` does not check the flag anymore